specification/communication/data-plane/Messages/stable/2024-02-01/communicationservicesmessages.json

```diff
@@ -183,23 +183,7 @@
                 "enum": [
                   "accepted",
                   "rejected"
-                ],
-                "x-ms-enum": {
-                  "name": "RepeatabilityResult",
-                  "modelAsString": false,
-                  "values": [
-                    {
-                      "name": "accepted",
-                      "value": "accepted",
-                      "description": "If the request was accepted and the server guarantees that the server state reflects a single execution of the operation."
-                    },
-                    {
-                      "name": "rejected",
-                      "value": "rejected",
-                      "description": "If the request was rejected because the combination of Repeatability-First-Sent and Repeatability-Request-ID were invalid\nor because the Repeatability-First-Sent value was outside the range of values held by the server."
-                    }
-                  ]
-                }
```

Comment:
I dont think your changes would affect this section. Not sure why this is getting taken out. Could you try to re-run the command?

Reply:
This is why the Lint check is failing btw.